### ISSUE TITLE
Why does this model not run in mobile Chrome and mobile Safari?

### ISSUE THREAD

**Issue by predict-r** [transcranial/keras-js#42]
Just made a toy example of an insult dector using a character level CNN: 
https://keras-predict-r.c9users.io/insults.html

The F1 score on the validation set is around 0.65, so it's not great but it works for many insults. It was trained on a small Kaggle insult dataset of around 3,500 examples. 

It will take up to 10 seconds to load the first time, and a text form field will appear. The form appears after the model Promise says the model is ready. 

In the mobile version, the problem is that never detects any insults that are detected in the non-mobile version on my laptop (desktop chrome and safari both work).

 The weights are around 10mb, so I'm guessing this has to do with memory, unless it could be a gpu issue.

**Comment by predict-r** [comment]
The script uses at maximum 12 mb [screenshot_robo](https://cloud.githubusercontent.com/assets/4193801/22088884/248c4bec-dd9c-11e6-9316-42585c0bfbf5.png)

Prediction times are pretty snappy given this quite large network with millions of parameters. About 60 ms.

**Comment by transcranial** [comment]
@predict-r Yeah, there are issues with mobile browsers, particularly on ios. Been meaning to investigate, but haven't had the time to yet.

**Comment by stekaiser** [comment]
I am also experiencing issues when trying to run the demos or one of my models on iOS with GPU enabled.

The error I am getting is `Bound framebuffer is not complete` at `keras.js:10533`. My guess is that it is a weblas related error (see https://github.com/waylonflinn/weblas/issues/30) but that is all I can contribute on this error.

@transcranial any ideas on this?
